### ISSUE TITLE
Dark Mode Feature

### ISSUE THREAD

**Issue by samosaboy** [Novusphere/discussions-app#1]
Currently the site does not support dark mode.

One of the implementation methods would be to use a body class "light" or "dark" and change styling accordingly.

**Comment by fixedelineation** [comment]
https://siasky.net/fACJh8hJ3vIhzrHXg8nhiErERILjdLiSKHQigb4O1ANL4g

Would this txt work for dark mode? @samosaboy

**Comment by fixedelineation** [comment]
[text.txt](https://github.com/Novusphere/discussions-app/files/4519911/text.txt) same as the above link.

**Comment by samosaboy** [comment]
No, how was this generated? The class names suffixes are dynamically created so hard coding them won't do anything. Our best bet would be to manually update all the styles for dark mode cases.

**Comment by fixedelineation** [comment]
https://nighteye.app/dark-css-generator/
